tests/fsharpqa/Source/Import/env.lst

```diff
@@ -83,11 +83,11 @@ NOMONO	SOURCE=reference2.fsx SCFLAGS="--nologo -r:reference1.dll" PRECMD="\$FSC_
 ###
 ### F# can consume FamOrAssembly and FamAndAssembly in combination with IVT
 ###
-    SOURCE=FamAndAssembly.fs         SCFLAGS="-a -r:AccessibilityTests.dll"    # FamAndAssembly.fs
+    SOURCE=FamAndAssembly.fs         SCFLAGS="-a -r:AccessibilityTests.dll" PRECMD="\$CSC_PIPE /debug /target:library /langversion:latest AccessibilityTests.cs"    # FamAndAssembly.fs
     ### See issue https://github.com/Microsoft/visualfsharp/issues/2496
-    ### SOURCE=FamOrAssembly.fs      SCFLAGS="-a -r:AccessibilityTests.dll"    # FamOrAssembly.fs
-    SOURCE=FamAndAssembly_NoIVT.fs   SCFLAGS="-a -r:AccessibilityTests.dll"    # FamAndAssembly_NoIVT.fs
-    ### SOURCE=FamOrAssembly_NoIVT.fs    SCFLAGS="-a -r:AccessibilityTests.dll"    # FamOrAssembly_NoIVT.fs
+    ### SOURCE=FamOrAssembly.fs      SCFLAGS="-a -r:AccessibilityTests.dll" PRECMD="\$CSC_PIPE /debug /target:library /langversion:latest AccessibilityTests.cs"    # FamOrAssembly.fs
```

Comment:
Can we not specify 7.2 explicit? Depending of the version of VS/MSBuild, latest could mean a version lower than 7.2 e.g. C# version 5 or 6.

Reply:
yes "latest" is not a good idea

Reply:
Speaking with @KevinRansom we're OK with `latest` so long as [`CSC_PIPE`](https://github.com/Microsoft/visualfsharp/blob/2d2104b1fce6ed0bae26695f28fb9913c9ff9bf5/tests/fsharpqa/Source/run.pl#L142) is updated to use `csc.exe` from the NuGet dependency I added in #4023 (with the caveat that the C# package might need to be upgraded; I don't know when 7.2 features were added.)

Currently I think `run.pl` grabs a version of `csc.exe` from the PATH instead of the explicit package version.

Reply:
@brettfo Should I just hardcode it in ``run.pl`` (then it would need to be adapted each time you update), or can it be passed down from somewhere?

Reply:
Ideally something passed in, even if that passed in value is hard-coded in `build.cmd`.

Looking through the code you could add a hard-coded argument in [`build.cmd`](https://github.com/Microsoft/visualfsharp/blob/ed152d7ba8c5d776581c7745a5217944dfb5fdfa/build.cmd#L952-L953) and then modify [`runall.pl`](https://github.com/Microsoft/visualfsharp/blob/ed152d7ba8c5d776581c7745a5217944dfb5fdfa/tests/fsharpqa/testenv/bin/runall.pl#L2095) to handle the new argument.  I didn't immediately find a way to pass through a value to `run.pl`, but it appears to prefer an [environment variable](https://github.com/Microsoft/visualfsharp/blob/ed152d7ba8c5d776581c7745a5217944dfb5fdfa/tests/fsharpqa/Source/run.pl#L140), so you could probably just do `$ENV{CSC_PIPE} = $passedInPathToCsc;` in `runall.pl`.

Reply:
It should be enough if I set CSC_PIPE from externally, right?

https://github.com/Microsoft/visualfsharp/blob/2d2104b1fce6ed0bae26695f28fb9913c9ff9bf5/tests/fsharpqa/Source/run.pl#L140-L143

Here it seems to read the env-var, and only defaults it to ``csc`` if it isn't already set.

Reply:
Good call, simpler is better.